Makefile

```diff
@@ -0,0 +1,42 @@
+MKFILE_PATH := $(abspath $(lastword $(MAKEFILE_LIST)))
+PROJECT_ROOT := $(dir $(MKFILE_PATH))
+TRANSCENDENCE_DEPS := $(shell find $(PROJECT_ROOT)/transcendence-app/ -type f -name "*.ts" | grep -v "node_module\|dist\|test")
```

Comment:
We could discuss which pattern brands a file that the generated files will depend on. Maybe more accurately than my first guess

Reply:
The idea of this env is something like ensuring we already have the deps installed and the project compiled?

Reply:
No, this is good old makefile prerequisites. If you attempt to run the transcendence-app/swagger-spec.yaml rule to generate the file of the same name, but the system sees the file and does not detect changes in the prerequisites, it will not run it.